projects/demo-integrations/cypress/tests/kit/input-range/input-range.spec.ts

```diff
@@ -234,6 +234,65 @@ describe('InputRange', () => {
                 .matchImageSnapshot('03-long-placeholder_pluralize');
         });
     });
+
+    describe('Mobile version', () => {
+        beforeEach(() => {
+            cy.viewport('iphone-x');
```

Comment:
can we use `cy.viewport('iphone-x')` inside `tuiVisit` if pseudoMobile enabled?

Reply:
I am not sure that it is a good idea to mix it.
It can causes problem if developer wants to use different viewport  (larger or smaller) with `pseudo`-mobile flag.